search-index/text2embeddings.py

```diff
@@ -18,10 +19,108 @@
 logger = get_logger(__name__)
 
 
+def get_text_from_list(text_block: dict, prev_processed_text_block: dict) -> str:
+    """
+    Format a list text block and prepend it to the previous text block, assuming it is the context of the list.
+    This is a fairly strong assumption, but works most of the time. Code to handle more complex cases has been started.
+
+    Args:
+        text_block (dict): text block dict.
+        prev_processed_text_block (dict): previous text block dict.
+
+    Returns:
+        str: Formatted list text block that is useful for indexing semantics.
+    """
+    text = text_block["custom_attributes"]["pretty_list_string"]
+    text = text.strip()
+    text = text.replace("*", "")  # Remove bullet points.
```

Comment:
I suggest we leave bullet points in as we'll display this processed text to the user on the search results page. If we decide it's better for NLP purposes to remove them, we can remove them for search purposes only in Elasticsearch using a [character filter](https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-pattern-replace-charfilter.html) and a [field](https://www.elastic.co/guide/en/elasticsearch/reference/current/multi-fields.html).